cloud-language-starter/src/main/java/com/sample/autoconfig/LanguageProperties.java

```diff
@@ -22,15 +24,9 @@ public class LanguageProperties implements CredentialsSupplier {
 
   private boolean useRest = false;
 
-  // retry settings: for each method, set *relevant* retry settings directly to its default value
-  // only writing out retry settings for one method (AnnotateText) as example
-  private Duration annotateTextInitialRetryDelay = Duration.ofMillis(100L); // do defaults here, or in autoconfig
-  private Double annotateTextRetryDelayMultiplier = 1.3;
-  private Duration annotateTextMaxRetryDelay = Duration.ofMillis(60000L);
-  private Duration annotateTextInitialRpcTimeout = Duration.ofMillis(600000L);
-  private Double annotateTextRpcTimeoutMultiplier = 1.0;
-  private Duration annotateTextMaxRpcTimeout = Duration.ofMillis(600000L);
-  private Duration annotateTextTotalTimeout = Duration.ofMillis(600000L);
+  // Configurable properties for overriding RetrySettings
+  // E.g. [prefix].language-service.retry.initial-retry-delay=PT0.5S
+  private Retry retry;
```

Comment:
Maybe `retrySettings` for `[prefix].language-service.retry-settings.initial-retry-delay=PT0.5S`

Reply:
Makes sense, will rename this variable

Reply:
If we want to support method level RetrySetting eventually, we may want to indicate that this a service level config. It's kind of implied at this moment but not that clear to me.

Reply:
Perhaps `serviceRetrySettings`? (Assuming method-level would look like e.g. `annotateTextRetrySettings`)